### PR TITLE
Sprint-2 userstory 456

### PR DESCRIPTION
**Pull request by navadeep0927**
please review my code

### CODE REVIEW

File: print-name.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 print("This python program reprints your name");
 name=input("Enter your name here: ");
 print("You have entered "+ name);
-#print("It is of type:"+ typeof(name));
+print("Lets find your lucky number");
+lucky_number=input("Enter your lucky number: ");
```
Comment (suryaval) [inline print-name.py:5]: please print out the lucky number

Comment (navadeep0927) [reply print-name.py:5]: incorporated review comments, please check

**Review by suryaval — APPROVED** [review]
looks good

**Review by suryaval — APPROVED** [review]
changes found

**Review by suryaval — APPROVED** [review]

**Pull request merged**